provider/resources.go

```diff
@@ -2190,6 +2191,664 @@ func Provider() tfbridge.ProviderInfo {
+					{Name: "SystemAdministrator", Value: "arn:aws:iam::aws:policy/job-function/SystemAdministrator"},
+					{Name: "VMImportExportRoleForAWSConnector", Value: "arn:aws:iam::aws:policy/service-role/VMImportExportRoleForAWSConnector"},
+					{Name: "ViewOnlyAccess", Value: "arn:aws:iam::aws:policy/job-function/ViewOnlyAccess"},
+				},
+			},
+			"aws:lambda/Runtime:Runtime": {
+				ObjectTypeSpec: schema.ObjectTypeSpec{
+					Type:        "string",
+					Description: "See https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html",
+				},
+				Enum: []*schema.EnumValueSpec{
+					{Value: "dotnetcore2.1", Name: "DotnetCore2d1"},
+					{Value: "dotnetcore3.1", Name: "DotnetCore3d1"},
+					{Value: "go1.x", Name: "Go1dx"},
+					{Value: "java8", Name: "Java8"},
+					{Value: "java8.al2", Name: "Java8AL2"},
+					{Value: "java11", Name: "Java11"},
+					{Value: "ruby2.5", Name: "Ruby2d5"},
+					{Value: "ruby2.7", Name: "Ruby2d7"},
+					{Value: "nodejs10.x", Name: "NodeJS10dX"},
+					{Value: "nodejs12.x", Name: "NodeJS12dX"},
+					{Value: "python2.7", Name: "Python2d7"},
+					{Value: "python3.6", Name: "Python3d6"},
+					{Value: "python3.7", Name: "Python3d7"},
+					{Value: "python3.8", Name: "Python3d8"},
+					{Value: "provided", Name: "Custom"},
+					{Value: "provided.al2", Name: "CustomAL2"},
+					{Value: "nodejs", Name: "NodeJS", DeprecationMessage: "No longer supported. New lambda functions created using this runtime will fail."},
+					{Value: "nodejs4.3-edge", Name: "NodeJS4d3Edge", DeprecationMessage: "No longer supported. New lambda functions created using this runtime will fail."},
+					{Value: "nodejs4.3", Name: "NodeJS4d3", DeprecationMessage: "No longer supported. New lambda functions created using this runtime will fail."},
+					{Value: "nodejs6.10", Name: "NodeJS6d10", DeprecationMessage: "No longer supported. New lambda functions created using this runtime will fail."},
+					{Value: "nodejs8.10", Name: "NodeJS8d10", DeprecationMessage: "No longer supported. New lambda functions created using this runtime will fail."},
+					{Value: "dotnetcore1.0", Name: "DotnetCore1d0", DeprecationMessage: "No longer supported. New lambda functions created using this runtime will fail."},
+					{Value: "dotnetcore2.0", Name: "DotnetCore2d0", DeprecationMessage: "No longer supported. New lambda functions created using this runtime will fail."},
+				},
+			},
+			"aws:route53/RecordType:RecordType": {
+				ObjectTypeSpec: schema.ObjectTypeSpec{
+					Type: "string",
+				},
+				Enum: []*schema.EnumValueSpec{
+					{Name: "A", Value: "A"},
+					{Name: "AAAA", Value: "AAAA"},
+					{Name: "CNAME", Value: "CNAME"},
+					{Name: "CAA", Value: "CAA"},
+					{Name: "MX", Value: "MX"},
+					{Name: "NAPTR", Value: "NAPTR"},
+					{Name: "NS", Value: "NS"},
+					{Name: "PTR", Value: "PTR"},
+					{Name: "SOA", Value: "SOA"},
+					{Name: "SPF", Value: "SPF"},
+					{Name: "SRV", Value: "SRV"},
+					{Name: "TXT", Value: "TXT"},
```

Comment:
Just curious: It looks like these have the same names and values. I see `aws:ssm/ParameterType:ParameterType` below only has values without names. Is there a reason `aws:route53/RecordType:RecordType` needs the names to be specified?

Reply:
Haha nope, just zoned out with all the copy-pasta. Will fix it up.